assets/js/main.js

```diff
@@ -65,38 +65,50 @@ $(document).ready(function() {
     /* ======= Modal Speaker ========= */
     // https://getbootstrap.com/docs/4.3/components/modal/#events
     $('#modal_speaker').on('show.bs.modal', function (e) {
-        if (! speakers) {
-            console.error('Data not found:', 'speakers');
+        var clearFields = ['job', 'company', 'social', 'description'];
+        var clearFieldsSelector = clearFields.reduce(function(selector, field) {
+            return (selector ? selector + ',' : '') + selectorSpeaker(field);
+        }, '');
+        var $link = $(e.relatedTarget);
+        var speakerCode = $link.data('speakerCode');  // from data-speaker-code, https://api.jquery.com/data/#data-html5
+        var speakerName = $link.data('speakerName');
+        var description = $link.data('description');
+        // if (description) description = $('<textarea />').html(description).text();  // unescape HTML entitles
+        $(clearFieldsSelector).empty();  // clear fields that may be missing in the data
+        if (! speakers || ! speakers[speakerCode]) {
+            console.error('Speaker not found:', speakerCode);
```

Comment:
Since this is not actually an error. Can we do a console.info and print something                    like. 'No speaker data present. Using Track data'